src/promptflow-evals/tests/unittests/test_f1_evaluator.py

```diff
@@ -0,0 +1,36 @@
+'''
+Created on Apr 12, 2024
+
+@author: nirovins
+'''
+import pytest
+import logging
+
+from unittest.mock import patch
+
+from promptflow.evals.evaluators.f1_score import F1ScoreEvaluator
+
+
+class TestF1ScoreEvaluator:

```

Comment:
Just a suggestion: We probably don't need a unit-test file for each evaluator.  If the intension here is only to test the logging, would suggest to rename the class to `TestEvaluatorLogging` or something similar. Same to the test file name.

Reply:
Fixed.